LEAF_Request_Portal/sources/Email.php

```diff
@@ -495,11 +495,14 @@ function setEmailToCcWithTemplate(string $tplLocation, bool $isCc = false): void
         $hasEmailTemplate = $this->getFilepath($tplLocation);
         $emailTemplate = __DIR__ . '/../templates/email/' . $hasEmailTemplate;
         if (file_exists($emailTemplate)) {
-            $emailListContentList =  explode(PHP_EOL, trim($this->setContent($emailTemplate)));
-            // For each line in template, add that email address, if valid
-            foreach($emailListContentList as $emailAddress) {
+            $emailContentList =  explode(PHP_EOL, trim($this->setContent($emailTemplate)));
+            $extractEmailReg = "/([a-z0-9\+_\-]+)(\.[a-z0-9\+_\-]+)*@([a-z0-9\-]+\.)+[a-z]{2,6}/i";
```

Comment:
Can we make this a class variable and assign the contents here and use it where needed instead of assigning it these three times? Also, is this one missing the $ before the /i? It is there for the other two.

Reply:
I think this is what github did to me/Shane yesterday (showing an older commit state).
This was an original approach that I changed after I realized there were issues with it.
The new version just grabs the trimmed line minus html and passes it to addRecipient and /or addCc.
They both use the updated regex, but it's only in those two places.
Could still update it, more a heads up that the preview is behind for some reason.

Reply:
I think he is talking about making the regex a class variable. This is also a variable set on line 221 and 286.

Reply:
I can still do that, just thought it was odd that the section of the comment is from a few commits ago.  This variable does not exist there anymore and had been gone by the time I PRd it.